CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # @digitalbazaar/ed25519-signature-2020 Changelog
 
+## 5.2.1 - 2023-03-TBD
+
+### Fixed
+- Fix url for Ed25519Signature2020 crypto suite spec in `README.md`.
```

Comment:
We don't need a new release or changelog entry for this as it doesn't affect the package / API in any way.

Reply:
Removed.